redbot/cogs/warnings/warnings.py

```diff
@@ -346,6 +346,12 @@ async def warn(
         if user.bot:
             await ctx.send(_("You cannot warn other bots."))
             return
+        if user == ctx.guild.owner:
+            await ctx.send(_("You cannot warn the server owner."))
+            return
+        if user.top_role >= ctx.author.top_role and ctx.author != ctx.guild.owner:
+            await ctx.send(_("The person you're trying to warn is equal or higher to you on the discord hierarchy, you cannot warn them."))
```

Comment:
I'm pretty sure it should be "is equal or higher than you in the discord hierarchy"